src/main/java/org/commcare/formplayer/application/FormController.java

```diff
@@ -244,7 +244,7 @@ public SubmitResponseBean submitForm(@RequestBody SubmitRequestBean submitReques
                 String response;
                 try {
                     response = submitService.submitForm(
-                            formEntrySession.getInstanceXml(),
+                            formEntrySession.getInstanceXml(false),
```

Comment:
@ctsims I noticed this when writing tests. This is the minimum change necessary to make sure we respect relevance when serializing the instance xml for this particular form submission. Just want to confirm this makes sense with you.

Reply:
Correct, this should result in the xml not including nodes which are irrelevant.